Makefile

```diff
@@ -102,4 +104,4 @@ CONTROLLER_GEN=$(shell which controller-gen)
 endif
 
 test-e2e:
-	./testing/validation.sh
\ No newline at end of file
+	./testing/validation.sh
```

Comment:
not exactly relevant to this PR but can you delete these targets - bundled in this PR since it doesn't have any side-effects